web/src/inspector.js

```diff
@@ -549,7 +549,17 @@ export class Inspector extends EventTarget {
       const get = (t) => t.getDateAdded();
       const first = get(torrents[0]);
       string = torrents.every((t) => get(t) === first)
-        ? new Date(first * 1000).toDateString()
+        ? new Date(first * 1000).toLocaleString(navigator.language, {
```

Comment:
```suggestion
        ? new Date(first * 1000).toLocaleString('en-US', {
```

Maybe this is better? On systems that is not English, this date will become the only non-English string and it might look out of place.